.github/workflows/presubmit.yml

```diff
@@ -311,14 +317,15 @@ jobs:
           if [[ ${{needs.check_files.outputs.run_job}} == false ]]; then
             exit 0
           fi
-          docker create --user dev --name taichi_test --gpus all -e DISPLAY=$DISPLAY -v /tmp/.X11-unix:/tmp/.X11-unix registry.taichigraphics.com/taichidev-ubuntu18.04:v0.1.1 /home/dev/unix_docker_test.sh $PY $GPU_TEST
+          docker create --user dev --name taichi_test --gpus all -e DISPLAY=$DISPLAY -v /tmp/.X11-unix:/tmp/.X11-unix registry.taichigraphics.com/taichidev-ubuntu18.04:v0.1.1 /home/dev/unix_docker_test.sh $PY $GPU_TEST $TI_WANTED_ARCHS
           docker cp .github/workflows/scripts/unix_docker_test.sh taichi_test:/home/dev/unix_docker_test.sh
           docker cp wheel/*.whl taichi_test:/home/dev/
           docker start -a taichi_test
         env:
           PY: py38
           GPU_TEST: ON
           DISPLAY: :1
+          TI_WANTED_ARCHS: "cuda,vulkan,opengl"
```

Comment:
let's also include cpu and cc here to make sure it works (since this is our release environment :P

Reply:
it seems that the GPU test's cc support is disabled in the cmake argument now, should we turn it on?

Reply:
let's give it a try? IIRC it should work